Items/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -1,417 +1,418 @@
-automatically hits armor. Targets take full damage to the torso, legs, and arms 
-(if armor is pierced or not present): (3*60 damage)+3xPlasma Burn within a 3m 
-radius. Only legs and torso (2*60 damage)+1xPlasma Burn within a 5m radius. At 
-least one hit must damage health to cause Plasma Burn.
+==============================
+MELEE WEAPONS & UNARMED FIGHTING
+==============================
+
+	These weapons/attacks use an 'Accuracy Modifier' instead of
+a 'Miss Chance', which is added to your combat dex modifier and
+and any relevant feat or skill bonuses to oppose your opponent's 
+Guard Class, or GC. Melee weapons do not pierce armor, unless otherwise 
+stated.
+
+	Melee weapons come in one of four size categories. The first 
+three are Light, One-handed, Two-handed. Characters start 
+with proficiency with some number of these based on their class. The 
+fifth category is Exotic. Acquiring proficiency with Exotic weapons does 
+not grant access to all Exotic weapons, but rather to a particular type, 
+such as whips. Exotic proficiencies may rarely be granted by Backgrounds, 
+and are more commonly acquired by feat.
+
+	When a Two-handed weapon uses a character's STR mod in its damage 
+calculation and the character's STR mod is positive, that character instead 
+uses 150% of their STR modifier to calculate damage. For example, if some 
+two-handed weapon dealt damage equal to 40 + 2xSTR nod and the character 
+wielding it had a STR mod of -4, each strike would deal 32 = 40 + (-4 * 2) 
+damage. A character with the same weapon and a STR mod of 12 would deal 
+76 = 40 + ((12 * 1.5) * 2) = 40 + (18 * 2) damage per strike.
+
+	Some One-handed weapons have the Hybrid size property. Characters 
+with proficiency with Two-handed melee weapons may make two-handed strikes 
+with these weapons, which allows them to add their STR modifier as if 
+the weapon was Two-handed (see the preceeding paragraph). Such a strike 
+must be declared to be two-handed before rolling to hit, and the character 
+doing so loses the benefits of any items that require his or her off hand to 
+function.*
+
+	Unarmed Strike is not a weapon and therefore does not have a 
+size category. All characters are considered proficient with Unarmed 
+Strike. Efficacy is an entirely separate issue.
+
+*As an example, a character with a STR mod of 8 is attacking an enemy with a 
+hammer of size One-handed - Hybrid that deals 40 + STR mod damage. Before 
+rolling the attack, the player may declare that the character is making a 
+two-handed strike, which will increase the damage from 40 + 8 = 48 to 
+40 + (8 * 1.5) = 40 + 12 = 52. If that character was holding a weapon or 
+defensive implement in his or her off-hand, that item cannot be used and 
+it provides no bonuses for one round.
+
+==============================
+UNARMED
+==============================
+
+== Unarmed Strike ==
+
+Damage: 20 + STR mod + DEX mod, minimum 10
```

Comment:
Can you explain the minimum 10 to me?

Reply:
An unarmed strike can never have outgoing damage less than 10. It's to prevent damage underflow from negative attribute mods.

Another option is to write a blanket rule that negative mods don't subtract damage from any melee weapon, but I rather like the idea that a 2 STR character is worse at swordplay than a 5 STR character.

Reply:
Honestly, besides putting the minimum above zero, very little conscious thought went into picking ten. I guess it's just a friend-shaped round number in a d10-based system.
But yes, nonzero minimum for basically that reason.